newsletter/components/content/HomeContent.jsx

```diff
@@ -71,7 +66,13 @@ export default function HomeContent() {
     }
   }
   useEffect(() => {
-    retrieveEmail()
+    // Connect (this is not cool becuase it pops up immediately for the user...)
+    // But this is currently the only way we can prompt the user to get their saved
+    // address. Also, this means the unlock status is loaded immediately
+    window.web3.currentProvider.enable()
+    if (lockState === 'unlocked') {
+      retrieveEmail()
```

Comment:
Since we now only retrieve the email when the user has unlocked the page, the email may remain undefined (see above)